### ISSUE TITLE
Python 3 seems not supported

### ISSUE THREAD

**Issue by math-a3k** [zuck/django-dropbox-storage#3]
Using:
```
Django==2.2.1
django-dropbox-storage==0.4.
dropbox==9.3.0
wagtail==2.5.1
```
When uploading an image file the following traceback is shown:
```
  File "/home/bot/code/conuruinca/venv/lib/python3.7/site-packages/django/db/models/fields/files.py", line 86, in save
    name = self.field.generate_filename(self.instance, name)
  File "/home/bot/code/conuruinca/venv/lib/python3.7/site-packages/django/db/models/fields/files.py", line 303, in generate_filename
    filename = self.upload_to(instance, filename)
  File "/home/bot/code/conuruinca/venv/lib/python3.7/site-packages/wagtail/images/models.py", line 48, in get_upload_to
    return instance.get_upload_to(filename)
  File "/home/bot/code/conuruinca/venv/lib/python3.7/site-packages/wagtail/images/models.py", line 129, in get_upload_to
    filename = self.file.field.storage.get_valid_name(filename)
  File "/home/bot/code/conuruinca/venv/lib/python3.7/site-packages/django/utils/functional.py", line 256, in inner
    self._setup()
  File "/home/bot/code/conuruinca/venv/lib/python3.7/site-packages/django/core/files/storage.py", line 363, in _setup
    self._wrapped = get_storage_class()()
  File "/home/bot/code/conuruinca/venv/lib/python3.7/site-packages/django/core/files/storage.py", line 358, in get_storage_class
    return import_string(import_path or settings.DEFAULT_FILE_STORAGE)
  File "/home/bot/code/conuruinca/venv/lib/python3.7/site-packages/django/utils/module_loading.py", line 17, in import_string
    module = import_module(module_path)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/bot/code/conuruinca/venv/lib/python3.7/site-packages/django_dropbox_storage/storage.py", line 8, in <module>
    from StringIO import StringIO
ModuleNotFoundError: No module named 'StringIO'
```
Seems that Python 3 is not supported by the package (https://stackoverflow.com/a/18284900).

If you don't update the code to Python 3, I think you should constrain the package requirement to Django<2.0 as from 2.0 the Python 2 support has been dropped from Django (and/or add a Python 2 tag)

**Comment by jmndao** [comment]
Hi, I've got the same issue while having dropbox for storage. The problem is dropbox is using an old version of python (python 2.x) therefore with python3.x the `from StringIO import StringIO` will lead to importError. The way you can solve it is to use `from io import StringIO` in the specified file `**'/home/bot/code/conuruinca/venv/lib/python3.7/site-packages/django_dropbox_storage/storage.py'**`, that is python3.x. For any other error you may encounter it may probably come from this. You can use this tip all along to overcome them all.

**Comment by zuck** [comment]
Fixed with e1df6cd41533dd5239706ba4cad36f7983e50521.
